apps/change-log.md

```diff
@@ -10,6 +10,18 @@ In order to keep track of these changes we recommend you to follow [@epagesdevs]
 
 <hr>
 
+## 2017-02-13
```

Comment:
Since the cart resource does not work anymore without the new cart token, we should mention this on the resource page (apps/api-reference/resource-carts) as well.

There is this text *A cart allows users to pick products from a shop and...*. Maybe this is a good place to mentions this? Otherwise it had to be added to each cart call which would be more effort...